l2geth/consensus/clique/scheduler.go

```diff
@@ -299,15 +340,18 @@ func (schedulerInst *Scheduler) schedulerRoutine() {
 	}
 }
 
+// handleChainHeadEventLoop checks whether the current block height reaches the currentStartMsg.maxHeight,
+// if it reaches the maxHeight, send batchIndex to channel batchDone.
 func (schedulerInst *Scheduler) handleChainHeadEventLoop() {
 	for {
 		select {
 		case chainHead := <-schedulerInst.chainHeadCh:
+			// Transactions of type QueueOriginL1ToL2 are not included in the batch
 			if chainHead.Block.Transactions().Len() != 0 && chainHead.Block.Transactions()[0].GetMeta() != nil && chainHead.Block.Transactions()[0].QueueOrigin() == types.QueueOriginL1ToL2 {
 				log.Debug("chainHead", "block_number", chainHead.Block.NumberU64(), "extra_data", hex.EncodeToString(chainHead.Block.Extra()))
 				continue
 			}
-			log.Info("schedulerInst handle chain head", "current_height", schedulerInst.blockchain.CurrentBlock().NumberU64(), "max_height", schedulerInst.currentStartMsg.MaxHeight)
+			log.Debug("schedulerInst handle chain head", "current_height", schedulerInst.blockchain.CurrentBlock().NumberU64(), "max_height", schedulerInst.currentStartMsg.MaxHeight)
```

Comment:
Was it nessary to log this each time with level info? @HaoyangLiu

Reply:
`debug` level is more reasonable.

Reply:
> 

LGTM